### PR TITLE
Change name from EmbyServer.dll to jellyfin.dll

### PR DESCRIPTION
**Pull request by Bond-009**
Update and remove some deps

### CODE REVIEW

File: MediaBrowser.Server.Mono/EmbyServer.csproj [hunk 1]
```diff
@@ -17,17 +18,11 @@
...
-    <PackageReference Include="sharpcompress" Version="0.22.0" />
-    <PackageReference Include="SimpleInjector" Version="4.3.0" />
-    <PackageReference Include="SkiaSharp" Version="1.60.3" />
+    <PackageReference Include="SkiaSharp" Version="1.68.0" />
     <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="1.1.11" />
     <PackageReference Include="SQLitePCLRaw.core" Version="1.1.11" />
     <PackageReference Include="SQLitePCLRaw.provider.sqlite3.netstandard11" Version="1.1.11" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.5.0" />
-    <Reference Include="Emby.Server.MediaEncoding">
-      <HintPath>..\ThirdParty\emby\Emby.Server.MediaEncoding.dll</HintPath>
```
Comment (JustAMan) [inline MediaBrowser.Server.Mono/EmbyServer.csproj]: I'm wary about this change, did we already replace it with compiled version?..

Comment (Bond-009) [reply MediaBrowser.Server.Mono/EmbyServer.csproj]: No, it’s just not used in this project.

**Comment by Bond-009** [comment]
As the only changes are minor updates and removing of unused dependencies nothing should break at runtime.

**Comment by Bond-009** [comment]
I bet the Dockerfile broke :)

**Comment by JustAMan** [comment]
@Bond-009 PR with Dockerfile fix please? :-)

**Pull request merged**